### ISSUE TITLE
Mhs35

### ISSUE THREAD

**Issue by slipk487** [tux1c/wavesharelcd-64bit-rpi#1]
Thanks so the touch seems to work I can see the console for a second before gui but stuck at black screen with _

**Comment by alexanderi96** [comment]
yeah same problem here, pi4b whith 64 bit raspberry pi os and 3.5" hat

**Comment by cppakko** [comment]
same problem,pi3b+ whith 64 bit kali

**Comment by JorgeMayoral** [comment]
I have the same problem with a Raspberry Pi 4 Model B running 64 bit Kali.

**Comment by waynew** [comment]
I get the same problem - note that the console *is* there - I can log in and it will flash, but then it will flash back to black.

I'm not super knowledgable about the framebuffers, but IIUC there's something going on where it's drawing the framebuffer that we want, and then flipping back to a blank one.

**Comment by waynew** [comment]
For me when it boots it does show all the console `[  OK  ]` messages. It's only when it flicks over to... whatever it's on, that it fails miserably.

**Comment by waynew** [comment]
Ooo! More promising info - for me:
```
def one():
    global r, g, b
    with open('/dev/fb0', 'wb') as framebuffer:
        for x in range(x_size):
            for y in range(y_size):
                r += 1
                r = r % 256

                g += 10
                g = g % 256

                b += 50
                b = b % 256
                pass
                framebuffer.write(r.to_bytes(1, byteorder='little'))
                framebuffer.write(g.to_bytes(1, byteorder='little'))
                framebuffer.write(b.to_bytes(1, byteorder='little'))
                framebuffer.write(b'\x00')
            if not x % 100:
                print(x)
```

This works! But if I do `/dev/fb1` instead, what I get is bands of color displayed, and then... not. So it looks like something is misconfigured to fight between fb0 and fb1

**Comment by alexstacey** [comment]
I got this working on a MHS35. See here for a script: https://github.com/alexstacey/MHS35-lcd-64bit-rpi

**Comment by greenpdx** [comment]
I tried to get it working by switching to X, but still doesn't work.
Raspberry Pi OS with desktop

    Release date: October 10th 2023
    System: 64-bit
    Kernel version: 6.1
    Debian version: 12 (bookworm)
    Size: 1,093MB
